lib/sidekiq_adhoc_job.rb

```diff
@@ -41,18 +41,23 @@ def self.strategies
   end
 
   class Configuration
-    attr_accessor :load_paths, :module_names, :strategy_name
+    attr_accessor :load_paths, :module_names, :strategy_name, :require_confirm_module_names
```

Comment:
```suggestion
    attr_accessor :load_paths, :module_names, :strategy_name, :require_confirm_worker_names
```

There's a difference in purpose between the original `module_names` and the new `require_confirm_x_names`. `module_names` allows configuring module names, as the name suggests, eg. `ActiveRecord::Base`, `ActiveRecord` will be the module name, and `ActiveRecord::Base` will be the worker name. When gem user configures `module_names = [ActiveRecord]`, the gem will only allow all workers under the `ActiveRecord` namespace to be listed.

Since the purpose of `require_confirm_x_names` is to configure the exact workers to require confirmation, it will be clearer to differentiate it with a `require_confirm_worker_names`.